src/Compilers/CSharp/Portable/BoundTree/Statement.cs

```diff
@@ -9,23 +9,52 @@
 
 namespace Microsoft.CodeAnalysis.CSharp
 {
-    internal partial class BoundStatement : IOperation
+    internal partial class BoundNode : IOperation, IOperationWithChildren
```

Comment:
@cston This is supposed to be a temporary internal interface that will exist until will we finish IOperation API design and implementation. Only the OperationWalker is aware about this interface to enable the analyzer driver to walk the descendants under unimplemented bound nodes. Having a special interface allows us to then just delete the interface and verify/fix up bound nodes still relying on it.